### PR TITLE
fix: logs download popup bug

### PR DESCRIPTION
**Pull request by liordino**
fix: added timestamp parameters for a date range on the gql queries for
machineLogs, serverLogs and transactions

feat: added optional limit and offset variables for the logs queries,
for filtering and pagination

feat: adapted the LogsDownloaderPopper to download the logs by whats set
on the filters

### CODE REVIEW

**Comment by liordino** [comment]
[Logs download popup bug](https://trello.com/c/wWsre81D/85-logs-download-popup-bug)

File: new-lamassu-admin/src/components/LogsDownloaderPopper.js [hunk 1]
```diff
@@ -153,39 +157,36 @@ const LogsDownloaderPopover = ({ name, getTimestamp, logs, title }) => {
...
     if (!range) return
 
-    if (range.from && !range.to) range.to = moment()
+    fetchLogs({
+      variables: {
+        ...args,
+        ...(selectedRadio === RANGE && {
+          ...(range.from && { from: range.from }),
+          ...(range.to && { until: range.to })
+        })
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/LogsDownloaderPopper.js]: this is pretty complicated to understand.

Better if you just create a empty object and add fields based on ifs or create something with _.fromPairs

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/components/LogsDownloaderPopper.js (hunk 1 above)
Comment (liordino) [reply new-lamassu-admin/src/components/LogsDownloaderPopper.js]: Done!

Comment (RafaelTaranto) [reply new-lamassu-admin/src/components/LogsDownloaderPopper.js]: @liordino I'm not sure if this new version would work if we pass down null in from and in to. Did you manage to test that?

Comment (liordino) [reply new-lamassu-admin/src/components/LogsDownloaderPopper.js]: You mean by selecting the ALL option on the radio button group and them passing null to the gql query? Yes, it's working.

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**